setup/setup.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+if [ "$(id -u)" -ne 0 ]; then 
+	echo "Please run w/ sudo." >&2; 
+	exit 1; 
+fi
+
+apt-get update
```

Comment:
highly recommend removing this and instead simply telling people to update their OS beforehand

Reply:
done